src/application/helpers.ts

```diff
@@ -396,4 +396,11 @@ export const prepareApplicationForSubmission = (sections: Record<string, any>):
     console.log(e);
     return null;
   }
-};
\ No newline at end of file
+};
+
+export const getRadioButtonInlineFieldChoicesSorted = (field: FormField, choices: FormFieldChoice[]): FormFieldChoice[] => {
```

Comment:
I would add a documentation comment here, saying what is the goal of the function in this case. Or maybe even better to rename the function to `getRadioButtonApplicantFieldChoicesSorted`, to make it clear from the outside that it is only intended for the applicant field and no other use cases.